### ISSUE TITLE
Justice helmets are invisible

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#26516]
(WEB REPORT BY: shiftyrail REMOTE: 206.221.180.138:7777)
# Revision
0a67788c71803328ed9acf9702c20a4677abbf75

# Description

They have no worn icon_state

**Comment by drawsstuff** [comment]
This is an injustice

**Comment by SonixApache** [comment]
i don't think we ever had sprites for them - i checked the history at some point
i know TG has sprites for them but they look retarded
